src/app/(proper_react)/(onboarding)/user/welcome/Onboarding.stories.tsx

```diff
@@ -14,5 +14,5 @@ export default meta;
 type Story = StoryObj<typeof OnboardingEl>;
 
 export const Onboarding: Story = {
-  render: () => <OnboardingEl user={{}} />,
+  render: () => <OnboardingEl user={{ email: "example@example.com" }} />,
```

Comment:
Sorry, this is an unrelated change I snuck in - I saw a red squiggly that this mock data did not include an email address.